system-x/services/fhir/src/main/java/software/tnb/fhir/service/Fhir.java

```diff
@@ -19,16 +19,19 @@ public abstract class Fhir extends ConfigurableService<NoAccount, NoClient, NoVa
     public Map<String, String> containerEnvironment() {
         return Map.of(
             "hapi.fhir.fhir_version", getConfiguration().fhirVersion(),
-            "hapi.fhir.reuse_cached_search_results_millis", "-1");
+            "DISABLE_AUTH", getConfiguration().getFhirAuthDisabled(),
+            "FHIR_USER", account().username(),
+            "FHIR_PWD", account().password());
     }
 
     @Override
     public String defaultImage() {
-        return "quay.io/fuse_qe/hapi:v7.2.0";
+        return "quay.io/fuse_qe/hapi-fhir-auth:v3.0.1";
```

Comment:
add a description to https://quay.io/repository/fuse_qe/hapi-fhir-auth?tab=info how the image was created

Reply:
decription added at https://quay.io/repository/fuse_qe/hapi-fhir-auth?tab=info